### PR TITLE
Added documentation about iin guard.

### PR DESCRIPTION
**Pull request by marcushanses**

### CODE REVIEW

File: docs/merchant/merchant/rules.md [hunk 1]
```diff
@@ -86,3 +89,30 @@ Spacing is important when writing a condition.
 When writing `20-12-24` is a string whereas `20 - 12 - 24` is a number. 
 
 Space can also be used as an AND-operator to chain several conditions together.
+
+### Extended State
+
+Depending on which [guards](./update#guards) are set on a merchant the [Pre-Authorization](../authorization/states.html#preauthorization), on which the rules apply, will contain additional information regarding the transaction under the field `authorization`.
```
Comment (robinnedergard) [inline docs/merchant/merchant/rules.md:95]:
```suggestion
Depending on which [guards](./update.html#guards) are set on a merchant the [Pre-Authorization](../authorization/states.html#preauthorization), on which the rules apply, will contain additional information regarding the transaction under the field `authorization`.
```

File: docs/merchant/merchant/update.md [hunk 2]
```diff
@@ -18,4 +18,19 @@ Authorization: Bearer <private.api.key> or <agent.api.key>
 
 The response is the updated merchant on the same form as in [create merchant](./create.html), in this case with the currency updated to SEK.
 
-Note that the [rules](./rules) can also be updated through a patch call.
\ No newline at end of file
+Note that the [rules](./rules) can also be updated through a patch call.
+
+## Guards
+To activate certain [guards](./rules#extended-state) PATCH the merchant with a list off all [guards](./rules#extended-state) that should be activated.
```
Comment (robinnedergard) [inline docs/merchant/merchant/update.md:24]:
```suggestion
To activate certain [guards](./rules.html#extended-state) PATCH the merchant with a list off all [guards](./rules.html#extended-state) that should be activated.
```

File: docs/merchant/merchant/update.md [hunk 3]
```diff
@@ -18,4 +18,19 @@ Authorization: Bearer <private.api.key> or <agent.api.key>
 
 The response is the updated merchant on the same form as in [create merchant](./create.html), in this case with the currency updated to SEK.
 
-Note that the [rules](./rules) can also be updated through a patch call.
\ No newline at end of file
+Note that the [rules](./rules) can also be updated through a patch call.
```
Comment (robinnedergard) [inline docs/merchant/merchant/update.md:21]:
```suggestion
Note that the [rules](./rules.html) can also be updated through a patch call.
```

**Review by robinnedergard — CHANGES_REQUESTED** [review]
Links not smooth

**Review by robinnedergard — APPROVED** [review]

**Review by nilssonemma — APPROVED** [review]

**Pull request merged**